// upload-csv/fesm5/upload-csv.js
import { Injectable, Component, NgModule, defineInjectable } from '@angular/core';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var UploadCsvService = /** @class */ (function () {
    function UploadCsvService() {
    }
    UploadCsvService.decorators = [
        { type: Injectable, args: [{
                    providedIn: 'root'
                },] },
    ];
    /** @nocollapse */
    UploadCsvService.ctorParameters = function () { return []; };
    /** @nocollapse */ UploadCsvService.ngInjectableDef = defineInjectable({ factory: function UploadCsvService_Factory() { return new UploadCsvService(); }, token: UploadCsvService, providedIn: "root" });
    return UploadCsvService;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var UploadCsvComponent = /** @class */ (function () {
    function UploadCsvComponent() {
    }
    /**
     * @return {?}
     */
    UploadCsvComponent.prototype.ngOnInit = /**
     * @return {?}
     */
    function () {
    };
    UploadCsvComponent.decorators = [
        { type: Component, args: [{
                    selector: 'lib-upload-csv',
                    template: "\n    <p>\n      upload-csv works!\n    </p>\n  ",
                    styles: []
                },] },
    ];
    /** @nocollapse */
    UploadCsvComponent.ctorParameters = function () { return []; };
    return UploadCsvComponent;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var UploadCsvModule = /** @class */ (function () {
    function UploadCsvModule() {
    }
    UploadCsvModule.decorators = [
        { type: NgModule, args: [{
                    imports: [],
                    declarations: [UploadCsvComponent],
                    exports: [UploadCsvComponent]
                },] },
    ];
    return UploadCsvModule;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

export { UploadCsvService, UploadCsvComponent, UploadCsvModule };
