// upload-csv/bundles/upload-csv.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('upload-csv', ['exports', '@angular/core'], factory) :
    (factory((global['upload-csv'] = {}),global.ng.core));
}(this, (function (exports,i0) { 'use strict';

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var UploadCsvService = (function () {
        function UploadCsvService() {
        }
        UploadCsvService.decorators = [
            { type: i0.Injectable, args: [{
                        providedIn: 'root'
                    },] },
        ];
        /** @nocollapse */
        UploadCsvService.ctorParameters = function () { return []; };
        /** @nocollapse */ UploadCsvService.ngInjectableDef = i0.defineInjectable({ factory: function UploadCsvService_Factory() { return new UploadCsvService(); }, token: UploadCsvService, providedIn: "root" });
        return UploadCsvService;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var UploadCsvComponent = (function () {
        function UploadCsvComponent() {
        }
        /**
         * @return {?}
         */
        UploadCsvComponent.prototype.ngOnInit = /**
         * @return {?}
         */
            function () {
            };
        UploadCsvComponent.decorators = [
            { type: i0.Component, args: [{
                        selector: 'lib-upload-csv',
                        template: "\n    <p>\n      upload-csv works!\n    </p>\n  ",
                        styles: []
                    },] },
        ];
        /** @nocollapse */
        UploadCsvComponent.ctorParameters = function () { return []; };
        return UploadCsvComponent;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var UploadCsvModule = (function () {
        function UploadCsvModule() {
        }
        UploadCsvModule.decorators = [
            { type: i0.NgModule, args: [{
                        imports: [],
                        declarations: [UploadCsvComponent],
                        exports: [UploadCsvComponent]
                    },] },
        ];
        return UploadCsvModule;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */

    exports.UploadCsvService = UploadCsvService;
    exports.UploadCsvComponent = UploadCsvComponent;
    exports.UploadCsvModule = UploadCsvModule;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
