// upload-csv/fesm2015/upload-csv.js
import { Injectable, Component, NgModule, defineInjectable } from '@angular/core';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class UploadCsvService {
    constructor() { }
}
UploadCsvService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] },
];
/** @nocollapse */
UploadCsvService.ctorParameters = () => [];
/** @nocollapse */ UploadCsvService.ngInjectableDef = defineInjectable({ factory: function UploadCsvService_Factory() { return new UploadCsvService(); }, token: UploadCsvService, providedIn: "root" });

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class UploadCsvComponent {
    constructor() { }
    /**
     * @return {?}
     */
    ngOnInit() {
    }
}
UploadCsvComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-upload-csv',
                template: `
    <p>
      upload-csv works!
    </p>
  `,
                styles: []
            },] },
];
/** @nocollapse */
UploadCsvComponent.ctorParameters = () => [];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class UploadCsvModule {
}
UploadCsvModule.decorators = [
    { type: NgModule, args: [{
                imports: [],
                declarations: [UploadCsvComponent],
                exports: [UploadCsvComponent]
            },] },
];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

export { UploadCsvService, UploadCsvComponent, UploadCsvModule };
